tests/postgres/alltypes_test.go

```diff
@@ -128,6 +129,41 @@ WHERE all_types.uuid = 'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11';
 	requireLogged(t, query)
 }
 
+func TestIntervalUpsert(t *testing.T) {
+	stmt := SELECT(Employee.AllColumns).FROM(Employee).
+		WHERE(Employee.EmployeeID.EQ(Int(1)))
+
+	var windy model.Employee
+	err := stmt.Query(db, &windy)
+	assert.Equal(t, windy.EmployeeID, int32(1))
+	assert.Equal(t, windy.FirstName, "Windy")
+	assert.Equal(t, windy.LastName, "Hays")
+	assert.Nil(t, err)
+	windy.PtoAccural = testutils.PtrOf("3h")
+	updateStmt := Employee.UPDATE(Employee.PtoAccural).SET(
+		Employee.PtoAccural.SET(CAST(String("3h")).AS_INTERVAL()),
+	).WHERE(Employee.EmployeeID.EQ(Int(1)))
+	_, err = updateStmt.Exec(db)
```

Comment:
updates and inserts should be executed inside transaction(and rollbacked) so we don't mess up test data. Check `testutils.ExecuteInTxAndRollback`